words/words.md

```diff
@@ -363,33 +363,33 @@ E' un paradosso. Il COVID-19 è estremamente contagioso, ma per fermarlo, dobbia
 
 [^exact_formula]: Ricorda che R = R<sub>0</sub> * il rapporto di contagi ancora permessi. Ricorda anche che il rapporto di contagi permessi = 1 - rapporto di contagi *impediti*.
 
-    Quindi, per avere R < 1, hai bisgno di avere R<sub>0</sub> * ContagiPermessi < 1.
+    Quindi, per arrivare a R < 1, serve che R<sub>0</sub> * ContagiPermessi < 1.
 
-    Quindi, ContagiPermessi < 1/R<sub>0</sub>
+    Quindi, che ContagiPermessi < 1/R<sub>0</sub>
 
-    Quindi, 1 - ContagiImpediti < 1/R<sub>0</sub>
+    Quindi, che 1 - ContagiImpediti < 1/R<sub>0</sub>
 
-    Qunidi, ContagiImpediti > 1 - 1/R<sub>0</sub>
+    Qunidi, che ContagiImpediti > 1 - 1/R<sub>0</sub>
 
-    Quindi, hai bisogno di impedire più di **1 - 1/R<sub>0</sub>** contagi per avere R < 1 e fermare il virus!
+    Quindi, è necessario impedire più di **1 - 1/R<sub>0</sub>** contagi per arrivare a R < 1, fermando il virus!
```

Comment:
again to keep translation "contain" -> "contenere" as in previous sentence we could change:

"per arrivare a R < 1, fermando il virus!" -> "per arrivare a R < 1 e contenere il virus!"